postgres/service.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: postgres
```

Comment:
Opinionated take: Should the service be `postgres` or `appname-postgres`?

I leaned towards `postgres` here because it becomes easier to reference in the app config, e.g. `DB_HOST=postgres`.